store/src/java/com/zimbra/cs/service/account/GetDistributionListMembers.java

```diff
@@ -183,7 +183,7 @@ private DLMembersResult getMembersFromLdap(Map<String, Object> context,
                 return getMembersFromLdap(account, group);
             } else {
                 // proxy to the home server of the group
-                ZimbraLog.account.debug("Proxying request to home server (%s) of group %s",
+                ZimbraLog.account.info("Proxying request to home server (%s) of group %s",
```

Comment:
Make more consistent with other similar-ish proxy requests?
Change
```    "Proxying request to home server (%s) of group %s"```
To something like?
```    "Proxying request: name=%s reason: target=%s", group.getName(), server.getName()```
  or
```    "Proxying request: dl.name=%s reason: target=%s", group.getName(), server.getName()```

side note: the notion of a "home server" for a DL is still so sad... and clearly needs to go away

Reply:
Done